tests/simple_http_resolver_ut.py

```diff
@@ -63,14 +63,14 @@ def set_responses(self):
                 responses.HEAD,
                 self.identifier_url,
                 status=200,
-                content_type='image/tiff'
+                content_type='image/tiff; charset=UTF-8'
         )
         responses.add(
                 responses.GET,
                 self.identifier_url,
                 body='II*\x00\x0c\x00\x00\x00\x80\x00  \x0e\x00\x00\x01\x03\x00\x01\x00\x00\x00\x01\x00\x00\x00\x01\x01\x03\x00\x01\x00\x00\x00\x01\x00\x00\x00\x02\x01\x03\x00\x01\x00\x00\x00\x08\x00\x00\x00\x03\x01\x03\x00\x01\x00\x00\x00\x05\x00\x00\x00\x06\x01\x03\x00\x01\x00\x00\x00\x03\x00\x00\x00\x11\x01\x04\x00\x01\x00\x00\x00\x08\x00\x00\x00\x15\x01\x03\x00\x01\x00\x00\x00\x01\x00\x00\x00\x16\x01\x03\x00\x01\x00\x00\x00\x08\x00\x00\x00\x17\x01\x04\x00\x01\x00\x00\x00\x04\x00\x00\x00\x1a\x01\x05\x00\x01\x00\x00\x00\xba\x00\x00\x00\x1b\x01\x05\x00\x01\x00\x00\x00\xc2\x00\x00\x00\x1c\x01\x03\x00\x01\x00\x00\x00\x01\x00\x00\x00(\x01\x03\x00\x01\x00\x00\x00\x02\x00\x00\x00@\x01\x03\x00\x00\x03\x00\x00\xca\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00H\x00\x00\x00\x01\x00\x00\x00H\x00\x00\x00\x01\x00\x00\x00\xff`\xe6q\x19\x08\x00\x00\x80\t\x00\x00\x80\n\x00\x00\x80\x0b\x00\x00\x80\x0c\x00\x00\x80\r',
                 status=200,
-                content_type='image/tiff'
+                content_type='image/tiff; charset=UTF-8'
```

Comment:
This means we're no longer testing the old behaviour – we know the header `image/tiff; charset=UTF-8` works, but not `image/tiff`. I'd rather have a test that covers both.

Can you add that, or would you rather I merge it and add the extra test myself?

Reply:
I thought that simple case is just one of sub-cases of complex-headers case. Here it tests with any valid `content-type` in it's generic sense, than to check all possible header values.

I will add special `simple` case quickly if it is required.

Reply:
at present, it has one `self.identifier_url`, around which test case runs. should i add another resource\_identifier, to do this check? I seems bit lost in figuring out convention test cases following. I fear, i may make it more intrusive. if you don't mind, can you write the test case best suited with conventions?